packages/manager/modules/pci/src/projects/project/data-processing/notebooks/notebook-details/notebook-details.scss

```diff
@@ -0,0 +1,21 @@
+.information-tile__definition {
+  display: flex;
+  color: #999;
+  font-size: 0.875rem;
+  font-weight: 400;
+  border-top: 1px solid #eaeaea;
+  padding: 0.813rem 0;
+}
+
+.notebook-title__header h2 {
+  margin: 0;
+  padding-top: 0.313rem;
+  line-height: normal;
+  color: #999;
+  font-size: 1rem;
+  font-weight: 400;
+}
+
+.notebook-details__content {
+  margin-top: 2rem;
+}
```

Comment:
**nitpick:** Avoid writing custom css. Try using utilities from bootstrap utilities and helpers from `ovh-ui-ki`
https://ovh.github.io/ovh-ui-kit/?path=/story/design-system-helpers-colors--primary
https://getbootstrap.com/docs/4.0/utilities/flex/
https://getbootstrap.com/docs/4.0/utilities/spacing/